tests/translate.service.spec.ts

```diff
@@ -117,6 +117,17 @@ describe('TranslateService', () => {
         mockBackendResponse(connection, '{}');
     });
 
+    it('should return they key when getting an object as the translation if supplying half of a key', () => {
+        translate.use('en');
+
+        translate.get('TEST').subscribe((res: string) => {
+            expect(res).not.toEqual({"SUBTEST": "Dit is een test"});
```

Comment:
I know both of these checks are not needed, but I did it to illustrate what it would have been before fixing the bug. If you want me to remove the not.toEqual check let me know, and I will!